src/test/script_segwit_tests.cpp

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2012-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <script/script.h>
+#include <test/setup_common.h>
```

Comment:
```suggestion
#include <test/util/setup_common.h>
```
This change makes the test works with current master branch.

Reply:
Good catch (I already had mostly forgotten about this PR).  I've rebased it to latest master now + added your fix, which indeed makes it work also for me.